vcx/docs/getting-started/getting-started.md

```diff
@@ -0,0 +1,646 @@
+﻿# Getting Started with Libindy
+
+## A Developer Guide for Building Indy Clients Using Libindy
+
+![logo](https://raw.githubusercontent.com/hyperledger/indy-node/master/collateral/logos/indy-logo.png)
+
+* [Getting Started with Libvcx](#getting-started-with-libvcx)
+  * [What Indy and Libindy are and Why They Matter](#what-indy-and-libindy-are-and-why-they-matter)
```

Comment:
> What Indy and Libindy are and Why They Matter

Can we re-name it to What Indy, Libindy, Libvcx are and Why They Matter

Reply:
Sure, missed this.